TextToSpeech/TextToSpeechImplementation.h

```diff
@@ -95,11 +97,13 @@ namespace Plugin {
         virtual uint32_t Configure(PluginHost::IShell* service) override;
         virtual void Register(Exchange::ITextToSpeech::INotification* sink) override ;
         virtual void Unregister(Exchange::ITextToSpeech::INotification* sink) override ;
+        virtual void RegisterWithCallsign(const string callsign,Exchange::ITextToSpeech::INotification* sink) override ;
```

Comment:
Should you also implement UnregisterWithCallsign()? I see that is mentioned as pure virtual function in the interface

Reply:
Unregisteration is handled in ` virtual void Unregister(Exchange::ITextToSpeech::INotification* sink) override ;`
it does not require a separate interface